### ISSUE TITLE
Server:Unstable appliance ISO apache2 fails to start

### ISSUE THREAD

**Issue by Conan-Kudo** [openSUSE/open-build-service#1552]
I downloaded the Feb 16 nightly appliance ISO and installed it. The installation was okay, but after installation, it failed to start apache2, and had errors.

Here's the systemd journal error information:
```
linux:~ # journalctl -u apache2
-- Logs begin at Wed 2016-02-17 08:28:53 UTC, end at Wed 2016-02-17 08:35:32 UTC. --
Feb 17 08:29:23 linux systemd[1]: Starting The Apache Webserver...
Feb 17 08:29:23 linux start_apache2[11939]: AH00526: Syntax error on line 64 of /etc/apache2/vhosts.d/obs.conf:
Feb 17 08:29:23 linux start_apache2[11939]: SSLCertificateFile: file '/srv/obs/certs/server.crt' does not exist or is empty
Feb 17 08:29:23 linux start_apache2[11961]: AH00526: Syntax error on line 64 of /etc/apache2/vhosts.d/obs.conf:
Feb 17 08:29:23 linux start_apache2[11961]: SSLCertificateFile: file '/srv/obs/certs/server.crt' does not exist or is empty
Feb 17 08:29:23 linux systemd[1]: Failed to start The Apache Webserver.
Feb 17 08:29:23 linux systemd[1]: Unit apache2.service entered failed state.
```

Here's line 64 of `/etc/apache2/vhosts.d/obs.conf`:

```
        SSLCertificateFile /srv/obs/certs/server.crt
```

Here's what it looks like in `/srv/obs/certs`:
```
linux:/srv/obs/certs # ls -hal --color
total 12K
drwx------  2 root   root   4.0K Feb 17 08:29 .
drwxr-xr-x 15 obsrun obsrun 4.0K Feb 17 08:29 ..
lrwxrwxrwx  1 root   root     11 Feb 17 08:29 server.crt -> server..crt
-rw-r--r--  1 root   root    887 Feb 17 08:29 server.key
```

It appears that server.crt points to an invalid location (`server..crt`), and that breaks httpd.

**Comment by M0ses** [comment]
This looks like empty FQHOSTNAME in this line:

https://github.com/openSUSE/open-build-service/blob/master/dist/setup-appliance.sh#L312

same root cause as in #1553 , but in this case we have to investigate why setup-appliance.sh does not exit on this error

**Comment by M0ses** [comment]
fixed with #1567
